src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryMath.cs

```diff
@@ -635,7 +635,7 @@ public static FormulaValue Abs(IRContext irContext, NumberValue[] args)
 
             if (arg0 == null)
             {
-                return new NumberValue(irContext, 0d);
+                throw new NotSupportedException();
```

Comment:
>throw new NotSupportedException(); [](http://example.com/codeflow?start=16&length=34)

Does this mean it's a bug in our code? Eg, we should have caught this already in binder/IR and never gotten here?

Reply:
If this case truly means we hit a bug, then we can probably just remove the if check altogether and accept the null ref on line 641.

Reply:
Correct, this should be caught in IR, so removed now.